www/_includes/css/_variables.scss

```diff
@@ -15,14 +18,21 @@ $spacing-horizontals: (
   large: $spacing-horizontal-large,
 );
 
-$mq-breakpoints: (
+$breakpoints: (
   small: $sizing-container-small,
   medium: $sizing-container-medium,
   large: $sizing-container-large,
   x-large: $sizing-container-xlarge,
   xx-large: $sizing-container-xxlarge,
 );
 
-$unquote-font-family-body: unquote($font-family-body);
-$unquote-font-family-display: unquote($font-family-display);
-$unquote-font-family-code: unquote($font-family-code);
+$show-breakpoints: (small, medium, large, x-large, xx-large);
+
+$unquote-font-family-body: string.unquote($font-family-body);
+$unquote-font-family-display: string.unquote($font-family-display);
+$unquote-font-family-code: string.unquote($font-family-code);
+
+@use "../../node_modules/sass-mq/mq" with (
```

Comment:
I forget; does this work in the Sass CLI? This may have been provided in some setups I’ve used. Keep as-is if not

```suggestion
@use "~/sass-mq/mq" with (
```

Reply:
It doesn't work on my end. Let's keep it as is 👍
![CleanShot 2025-09-17 at 23 42 19@2x](https://github.com/user-attachments/assets/55f910bf-851e-48e5-88d6-fea8da555c16)